### PR TITLE
Fix critical and major bugs from Sonar in JS files

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes the critical and major bugs detected in JS files in Sonar. I have removed trailing spaces in the touched files too.

### CODE REVIEW

File: src/js/tabs/servos.js [hunk 1]
```diff
@@ -57,9 +57,9 @@ TABS.servos.initialize = function (callback) {
         $('div.tab-servos table.fields tr.main').append(servoHeader);
 
         function process_servos(name, alternate, obj) {
```
Comment (mikeller) [inline src/js/tabs/servos.js]: `alternate` is unused as well.

Comment (McGiverGim) [reply src/js/tabs/servos.js]: Good catch. Fixed in a new commit. With this commit I have fixed too some others Sonar complaints in the same file... to not upload it alone :)

**Comment by McGiverGim** [comment]
FIxed as per review...

**Comment by McGiverGim** [comment]
Rebased against master...

**Pull request merged**